src/components/erc721/collectibles/collectibles_card_list.tsx

```diff
@@ -66,11 +72,20 @@ interface Props {
     sortType: CollectibleSortType;
     filterType: CollectibleFilterType;
     limit?: number;
+    isLoading?: boolean;
```

Comment:
PP: I think this doesn't need to be optional (at least I removed the `?` and it compiled fine).

Reply:
Yup. Made it optiotional in case you are feeding the component with a collection in a "sync way" and you don't want/need to signal any loading at all.